### ISSUE TITLE
Back out WORD_SIZE patch

### ISSUE THREAD

**Issue by lopopolo** [artichoke/mspec#1]
Once https://github.com/artichoke/artichoke/pull/212 lands, remove patch 9a9d3d0a2339a8cb3a0579d7cac6d76ce5870ab3 and revendor mspec into [artichoke/spec-runner](https://github.com/artichoke/artichoke/tree/035c285c69c2325d6b7f552c0480b892014ab994/spec-runner/vendor/mspec).

**Comment by lopopolo** [comment]
removed and force-pushed

**Comment by lopopolo** [comment]
merged into artichoke/artichoke.
